Makefile

```diff
@@ -597,10 +608,10 @@ cover_profile:
 	$Q for dir in $(PKG_TEST_DIRS); do \
 		mkdir -p $(BUILD)/"$$dir"; \
 		go test "$$dir" $(TEST_ARG) -coverprofile=$(BUILD)/"$$dir"/coverage.out || exit 1; \
-		cat $(BUILD)/"$$dir"/coverage.out | grep -v "^mode: \w\+" >> $(UNIT_COVER_FILE); \
+		(cat $(BUILD)/"$$dir"/coverage.out | grep -v "^mode: \w\+" >> $(UNIT_COVER_FILE)) || true; \
```

Comment:
What is the grep filter for?  the fact that multiline comments are hard would make me lean to towards throwing into a bash script

Reply:
Historically, it was because you can just concat multiple coverage files to have go  transparently compute the combined coverage... except for those mode lines, which would cause it to error.

Nowadays I know there are better options in a few ways, but I haven't dug into them in detail yet.

---

Re script file: yeah, this is borderline or slightly past I think.  But it is hopefully temporary and will also-hopefully be simpler after a modernization pass, so I'm kinda reluctant to do more than the minimum / putting it in here keeps the badness visible until it's replaced / etc :/

I could be convinced to do otherwise, just didn't feel like it was worth it at the moment.